CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - `bitmovin-player@^8.31.0` as peer dependency.
 - Possibility to set detailed Conviva Device Metadata via `ConvivaAnalyticsConfiguration.deviceMetadata`
+- Possibility to override values for internally set custom metdata keys streamType, playerType and vrContentType
```

Comment:
I'd suggest putting code in backticks to make it obvious that those are the actual names from the code:
```suggestion
- Possibility to override values for internally set custom metdata keys `streamType`, `playerType` and `vrContentType`
```

Reply:
Done.